llama_cpp_guidance/llm.py

```diff
@@ -50,6 +50,8 @@ def __init__(
         role_end_tag="<|im_end|>",
         chat_mode=False,
         seed: int = 0,
+        role_to_name: Dict[str, str] = {},
+        **llama_kwargs: Dict[str, Any]
```

Comment:
I like this idea! Alas, this argument is not being passed to `Llama`. Perhaps line 74 should have `**llama_kwargs`, too?

Reply:
Oh oops, my bad!